### ISSUE TITLE
spinner corrupted when use Promise.all and each promise has a spinner

### ISSUE THREAD

**Issue by chaoyangnz** [sindresorhus/ora#173]
I use a spinner for each promise, and when I run `Promise.all` and hope promises are happening in parallel. 

Ora outputs corrupted texts.

**Comment by sindresorhus** [comment]
Duplicate of #116

**Comment by sindresorhus** [comment]
Ora currently only supports one spinner at the time.

**Comment by LaboratorioInternacionalWeb** [comment]
You can try: [https://www.npmjs.com/package/multispinners](https://www.npmjs.com/package/multispinners)
